### DISCUSSION TITLE
Generate test coverage report

### DISCUSSION THREAD

**Discussion by alexalex1810** [JamalZeynalov/swagger-coverage-py#19] [Q&A] [answered]
How can I get report with test results? Shall I run command swagger-coverage-commnadline?

**Comment by JamalZeynalov** [comment, marked as answer]
@alexalex1810 Hi,
The report will be generated automatically. You just need to follow all steps described in Readme.
This fixture is responsible for the report generation:
https://github.com/JamalZeynalov/swagger-coverage-py#3-add-the-session-scoped-fixture
